src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "name": "HSLU simple MEP results",
-  "version": "2.0.15",
+  "version": "2.1.0",
```

Comment:
fräch!

Reply:
Just followed the README-Development-Create a new release... 😅  tued mer sorry, hehe